tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -227,3 +229,121 @@ def forward_xp(self, inputs, xp):
         a, b = inputs
         out = xp.linalg.solve(a, b)
         return out,
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    chainer.testing.product({
+        'shape': [(1, 1), (2, 3), (3, 2), (6, 6)],
+        'dtype': ['float32', 'float64'],
+        'full_matrices': [False],
+        'compute_uv': [True]
+    }) + chainer.testing.product({
+        'shape': [(1, 1), (2, 3), (3, 2), (6, 6)],
+        'dtype': ['float32', 'float64'],
+        'full_matrices': [True],
+        'compute_uv': [False],
+        'skip_backward_test': [True],
+        'skip_double_backward_test': [True],
+    })
+))
+class TestSVD(NumpyLinalgOpTest):
+
+    def generate_inputs(self):
+        a = numpy.random.random(self.shape).astype(self.dtype)
+        return a,
+
+    def forward_xp(self, inputs, xp):
+        a, = inputs
+        out = xp.linalg.svd(a,
+                            full_matrices=self.full_matrices,
+                            compute_uv=self.compute_uv)
+        # NOTE: cuSOLVER's (CuPy's) and NumPy's outputs of u and v might
+        # differ in signs, which is not a problem mathematically
```

Comment:
For the record, the tests in CuPy has been strengthened (for complex numbers) https://github.com/cupy/cupy/pull/2338. Mostly wanted to leave that information accessible from here but we could leave a reference to this PR in the comment. Not a strong opinion.